src/routes/auth.js

```diff
@@ -1,6 +1,38 @@
 const express = require('express');
 const authController = require('../controllers/auth');
+const { passport } = require('../config/passport');
 
 const router = express.Router();
 
+/**
+ *  @swagger
```

Comment:
Noor created a .yaml file for swagger, so I don't think you should write the documentation in routes because it ignores the single responsibility. Routes should only be responsible for routing.

Reply:
Alright, I'll try to implement it this way